test_scripts/WebEngine/WebEngineProjection/014_HMI_status_transitions_different_applications_interactions.lua

```diff
@@ -0,0 +1,184 @@
+    [3] = { type = "DEFAULT", isMedia = false, device = devices.default },
+    [4] = { type = "NAVIGATION", isMedia = true, device = devices.default }
+  },
+  steps = {
+    [1] = {
+      action = { event = common.userActions.activateApp, appId = 1 },
+      checks = {
+        onHmiStatus = {
+          [1] = { hmiLvl = "FULL", audio = "AUDIBLE", video = "NOT_STREAMABLE" },
+          [2] = { }, -- hmiLvl = "NONE", audio = "NOT_AUDIBLE", video = "NOT_STREAMABLE"
+          [3] = { }, -- hmiLvl = "NONE", audio = "NOT_AUDIBLE", video = "NOT_STREAMABLE"
+          [4] = { }  -- hmiLvl = "NONE", audio = "NOT_AUDIBLE", video = "NOT_STREAMABLE"
+        }
+      }
+    },
+    [2] = {
+      action = { event = common.userActions.activateApp, appId = 2 },
+      checks = {
+        onHmiStatus = {
+          [1] = { hmiLvl = "LIMITED", audio = "AUDIBLE", video = "NOT_STREAMABLE" },
+          [2] = { hmiLvl = "FULL", audio = "NOT_AUDIBLE", video = "STREAMABLE" },
+          [3] = { }, -- hmiLvl = "NONE", audio = "NOT_AUDIBLE", video = "NOT_STREAMABLE"
+          [4] = { }  -- hmiLvl = "NONE", audio = "NOT_AUDIBLE", video = "NOT_STREAMABLE"
+        }
+      }
+    },
+    [3] = {
+      action = { event = common.userActions.activateApp, appId = 3 },
+      checks = {
+        onHmiStatus = {
+          [1] = { }, -- hmiLvl = "LIMITED", audio = "AUDIBLE", video = "NOT_STREAMABLE"
+          [2] = { hmiLvl = "LIMITED", audio = "NOT_AUDIBLE", video = "STREAMABLE" },
+          [3] = { hmiLvl = "FULL", audio = "NOT_AUDIBLE", video = "NOT_STREAMABLE" },
+          [4] = { } -- hmiLvl = "NONE", audio = "NOT_AUDIBLE", video = "NOT_STREAMABLE"
+        }
+      }
+    },
+    [4] = {
+      action = { event = common.userActions.activateApp, appId = 4 },
+      checks = {
+        onHmiStatus = {
+          [1] = { }, -- hmiLvl = "LIMITED", audio = "AUDIBLE", video = "NOT_STREAMABLE"
+          [2] = { hmiLvl = "BACKGROUND", audio = "NOT_AUDIBLE", video = "NOT_STREAMABLE" },
+          [3] = { hmiLvl = "BACKGROUND", audio = "NOT_AUDIBLE", video = "NOT_STREAMABLE" },
+          [4] = { hmiLvl = "FULL", audio = "AUDIBLE", video = "STREAMABLE" }
+        }
+      }
+    },
+    [5] = {
+      action = { event = common.userActions.phoneCallStart, appId = "none" },
+      checks = {
+        onHmiStatus = {
+          [1] = { hmiLvl = "BACKGROUND", audio = "NOT_AUDIBLE", video = "NOT_STREAMABLE" },
+          [2] = { }, -- hmiLvl = "BACKGROUND", audio = "NOT_AUDIBLE", video = "NOT_STREAMABLE"
+          [3] = { }, -- hmiLvl = "BACKGROUND", audio = "NOT_AUDIBLE", video = "NOT_STREAMABLE"
+          [4] = { hmiLvl = "LIMITED", audio = "NOT_AUDIBLE", video = "STREAMABLE" }
+        }
+      }
+    },
+    [6] = {
+      action = { event = common.userActions.phoneCallEnd, appId = "none" },
+      checks = {
+        onHmiStatus = {
+          [1] = { hmiLvl = "LIMITED", audio = "AUDIBLE", video = "NOT_STREAMABLE" },
```

Comment:
How can two apps be now audible? Should app 1 be background because app 4 is media?

Reply:
@kshala-ford  According requirements introduced in [Enhancing onHMIStatus with a New Parameter for Video Streaming State](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0150-video-streaming-state.md) proposal two applications can be in `AUDIBLE` state in some cases.
Current SDL behavior is described in the item 4.2. of second table from the [Proposed solution](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0150-video-streaming-state.md#proposed-solution)

Reply:
Summary paragraph from the proposal:

> *In the case of co-existence of a NAVIGATION app and a MEDIA or COMMUNICATION app, when the NAVIGATION app does not start audio steaming service, the MEDIA/COMMUNICATION app is AUDIBLE; When the NAVIGATION app starts audio streaming service, the MEDIA/COMMUNICATION app is either ATTENUATED if the system supports mixing or NOT_AUDIBLE if the system does not support mixing.


In the test the navigation app has not started the audio service, so both apps with audible status is ok.